src/content/docs/en/reference/api-reference.mdx

```diff
@@ -1116,6 +1116,10 @@ The `getStaticPaths()` function should return an array of objects to determine w
 
 It can also be used in static file endpoints for [dynamic routing](/en/guides/endpoints/#params-and-dynamic-routing).
 
+:::tip
+When using TypeScript, use the [`GetStaticPaths`](/en/guides/typescript/#infer-getstaticpaths-types) type utility to ensure type-safe access of your `params` and `props`.
+:::
```

Comment:
```suggestion
```
I agree with the comment from the issue about NOT putting this here. This is about `getStaticPaths()` and its type should be documented there.

Also, it's not a "tip" if it's "how things work."  😄